pkg/registry/core/project/strategy.go

```diff
@@ -151,3 +156,50 @@ func (projectStatusStrategy) PrepareForUpdate(ctx context.Context, obj, old runt
 func (projectStatusStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object) field.ErrorList {
 	return validation.ValidateProjectStatusUpdate(obj.(*core.Project), old.(*core.Project))
 }
+
+// ToSelectableFields returns a field set that represents the object
+// TODO: fields are not labels, and the validation rules for them do not apply.
+func ToSelectableFields(project *core.Project) fields.Set {
+	// The purpose of allocation with a given number of elements is to reduce
+	// amount of allocations needed to create the fields.Set. If you add any
+	// field here or the number of object-meta related fields changes, this should
+	// be adjusted.
+	projectSpecificFieldsSet := make(fields.Set, 3)
+	projectSpecificFieldsSet[core.ProjectNamespace] = getNamespace(project)
+	return generic.AddObjectMetaFieldsSet(projectSpecificFieldsSet, &project.ObjectMeta, true)
```

Comment:
`Project` is not namespaced.
```suggestion
	projectSpecificFieldsSet := make(fields.Set, 2)
	projectSpecificFieldsSet[core.ProjectNamespace] = getNamespace(project)
	return generic.AddObjectMetaFieldsSet(projectSpecificFieldsSet, &project.ObjectMeta, false)
```